ldk/javascript/src/whisper/mapper.ts

```diff
@@ -112,6 +112,13 @@ export function mapToInternalChildComponent(
     case WhisperComponentType.Markdown:
     case WhisperComponentType.Message:
       return component;
+    case WhisperComponentType.DropZone:
+      return {
+        ...component,
+        onDrop: (error, param, whisper) => {
```

Comment:
Do we need to set a 'path' value in the state map so it will be returned back to loop author as part of any onSelect/onChange/onDrop events?

Reply:
Maybe! TBH I'm not sure how we'll handle the state of this specific component. Unlike our other state values, these file objects are representations of something the user can kinda interact with, and not the actual value. But this will be handled in HELPS-854 and HELPS-856.

Reply:
There is a 'path: string;' which is returned. Perhaps we could set this in the sitemap? Was wondering if we are going to send no value back with the stateMap to the loop author, then our direction on Form Submit is not going to be correct, as they wont be able to validate it. Thoughts?

Reply:
These are great questions and ideas...  I will spend more time thinking about this... as part of HELPS-854.